src/import/codegen.ts

```diff
@@ -45,12 +45,18 @@ export function emitHeader(code: CodeMaker, custom: boolean) {
   code.line();
 }
 
+export function getTypeName(basename: string, version: string) {
+  // add an API version postfix only if this is core API (`import k8s`).
```

Comment:
I don't believe this is working as described - not sure if intentional or not.

I checked out your PR, built it, and used it to import these CRDs in a project: https://github.com/jetstack/cert-manager/releases/download/v1.2.0/cert-manager.crds.yaml

Previously, it generated `Certificate` but now it generates `CertificateV1Alpha2`.

I don't think this is necessarily blocking (I think we discussed that fixing https://github.com/cdk8s-team/cdk8s-cli/issues/28 might require separate changes) but this comment should probably be updated.

Reply:
Good catch. Fixed to apply to k8s resources.